### PR TITLE
Proper handling of spectral resolution

### PR DESCRIPTION
**Pull request by andamian**
Fix for #1816 

Both (m)
`Alma.query_async(payload={'em_resolution': 6.929551916151968e-05})`
and (KHz)
`Alma.query_async(payload={'spectral_resolution':2000000})`
work

Column `spectral_resolution` is also available from direct TAP queries.

### CODE REVIEW

File: astroquery/alma/tests/test_alma.py [hunk 1]
```diff
@@ -199,17 +199,6 @@ def test_gen_datetime_sql():
         common_select + "(58849.0<=t_min AND t_min<=58880.0)"
 
 
-def test_gen_spec_res_sql():
-    common_select = 'select * from ivoa.obscore WHERE '
-    assert _gen_sql({'spectral_resolution': 70}) == common_select + "em_resolution=20985472.06"
```
Comment (bsipocz) [inline astroquery/alma/tests/test_alma.py:204]: I don't see any tests now that either has `em_resolution` or `spectral_resolution` in them, could you please add some test or docs example coverage?

Comment (andamian) [reply astroquery/alma/tests/test_alma.py:204]: The specific transformation function was replaced with the simpler/tested `_gen_numerci_sql` function so I thought I might get away without adding extra tests. But it was easy enough to do it.

Comment (bsipocz) [reply astroquery/alma/tests/test_alma.py:204]: Yes, it maybe an overkill. 

I think what I would like for all modules is to have at least one test/doc example where they run a query that uses all the possible parameters and actually run that query. Just to be on the extra safe side and notice when the server changes something. But this is beyond the scope here.

**Review by bsipocz — APPROVED** [review]

**Comment by bsipocz** [comment]
Thanks!

**Comment by andamian** [comment]
No. Thank you!

**Pull request merged**